mc2mc/internal/client/odps.go

```diff
@@ -21,6 +21,48 @@ type odpsClient struct {
 	isDryRun               bool
 }
 
+var (
```

Comment:
can we move this implementation to other file? client/util.go should be sufficient